src/data/item-desc.csv

```diff
@@ -1,71 +1,71 @@
-"text", "translation"
-"短距離の装備に定評のあるメーカーの\n話題作。覚醒Lv強化で使用できる。",""
-"マイルシューズ界を席巻したロング\nセラー製品。覚醒Lv強化で使用できる。",""
-"「中距離シューズの大本命！」でおなじみ。\n覚醒Lv強化で使用できる。",""
-"長距離でも疲れにくい「ド軽量」シリーズ。\n覚醒Lv強化で使用できる。",""
-"ダートでも汚れない「土踏まず」\nブランド製。覚醒Lv強化で使用できる。",""
-"覚醒Lv強化に必要となる素材",""
-"飲んで元気を取り戻そう！\n使用するとTPが30回復する。",""
-"1秒チャージで2時間キープ！？\n使用するとRPが1回復する。",""
-"ベテラントレーナー御用達！\n使用するとRPが5回復する。",""
-"育成ウマ娘ガチャが引けるチケット。\nガチャの一覧から使用できる。",""
-"虹色に輝く不思議な力をまとった\nニンジン型の宝石。",""
-"ウマ娘が夢を掴んだ証とされる\n煌めく力。覚醒Lv強化で使用できる。",""
-"ルーレットダービー専用コイン。\nイベントを遊ぶことで入手できる。",""
-"蹄鉄型の銀色のお守り。\nショップで様々なものと交換できる。",""
-"蹄鉄型の金色のお守り。\nショップで様々なものと交換できる。",""
-"蹄鉄型の虹色のお守り。\nショップで様々なものと交換できる。",""
-"幸運を呼ぶ四つ葉のクローバー。\nショップで様々なものと交換できる。",""
-"イベントを遊ぶことで入手できる。\n集めることで様々な報酬を獲得できる。",""
-"備えがあればうれしいな。\nトレセン学園の様々な場面で使用できる。",""
-"レース前にウマ娘の能力の分析ができる",""
-"ダートGⅠ・フェブラリーSを優勝した証。\n覚醒Lv強化で使用できる。",""
-"短距離GⅠ・高松宮記念を優勝した証。\n覚醒Lv強化で使用できる。",""
-"中距離GⅠ・大阪杯を優勝した証。\n覚醒Lv強化で使用できる。",""
-"クラシック級GⅠ・桜花賞を優勝した証。\n覚醒Lv強化で使用できる。",""
-"クラシック級GⅠ・皐月賞を優勝した証。\n覚醒Lv強化で使用できる。",""
-"長距離GⅠ・天皇賞（春）を優勝した証。\n覚醒Lv強化で使用できる。",""
-"クラシック級GⅠ・NHKマイルCを優勝\nした証。覚醒Lv強化で使用できる。",""
-"マイルGⅠ・ヴィクトリアマイルを\n優勝した証。覚醒Lv強化で使用できる。",""
-"クラシック級GⅠ・オークスを優勝した証。\n覚醒Lv強化で使用できる。",""
-"クラシック級GⅠ・日本ダービーを\n優勝した証。覚醒Lv強化で使用できる。",""
-"マイルGⅠ・安田記念を優勝した証。\n覚醒Lv強化で使用できる。",""
-"中距離GⅠ・宝塚記念を優勝した証。\n覚醒Lv強化で使用できる。",""
-"短距離GⅠ・スプリンターズSを\n優勝した証。覚醒Lv強化で使用できる。",""
-"クラシック級GⅠ・秋華賞を優勝した証。\n覚醒Lv強化で使用できる。",""
-"クラシック級GⅠ・菊花賞を優勝した証。\n覚醒Lv強化で使用できる。",""
-"中距離GⅠ・天皇賞（秋）を優勝した証。\n覚醒Lv強化で使用できる。",""
-"マイルGⅠ・マイルCSを優勝した証。\n覚醒Lv強化で使用できる。",""
-"中距離GⅠ・エリザベス女王杯を\n優勝した証。覚醒Lv強化で使用できる。",""
-"中距離GⅠ・ジャパンカップを優勝した証。\n覚醒Lv強化で使用できる。",""
-"ダートGⅠ・チャンピオンズカップを\n優勝した証。覚醒Lv強化で使用できる。",""
-"ジュニア級GⅠ・阪神JFを優勝した証。\n覚醒Lv強化で使用できる。",""
-"ジュニア級GⅠ・朝日杯FSを優勝した証。\n覚醒Lv強化で使用できる。",""
-"長距離GⅠ・有馬記念を優勝した証。\n覚醒Lv強化で使用できる。",""
-"ジュニア級GⅠ・ホープフルSを\n優勝した証。覚醒Lv強化で使用できる。",""
-"大事なレース日に遅刻は厳禁！育成の\n目標レースでコンティニューができる。",""
-"デイリーレースの挑戦に必要なチケット。",""
-"レジェンドレースの挑戦に必要なチケット。",""
-"トレーナー同士の友情の証。\nショップで様々なものと交換できる。",""
-"クラシック級GⅠ・ジャパンダートダービー\nを優勝した証。覚醒Lv強化で使用できる。",""
-"ダートGⅠ・JBCスプリントを優勝した証。\n覚醒Lv強化で使用できる。",""
-"ダートGⅠ・JBCクラシックを優勝\nした証。覚醒Lv強化で使用できる。",""
-"ダートGⅠ・JBCレディスクラシックを\n優勝した証。覚醒Lv強化で使用できる。",""
-"ダートGⅠ・帝王賞を優勝した証。\n覚醒Lv強化で使用できる。",""
-"ダートGⅠ・東京大賞典を優勝した証。\n覚醒Lv強化で使用できる。",""
-"能力を引き出す不思議な力。\nサポートカードのLv強化で使用できる。",""
-"サポートカードガチャが引けるチケット。\nガチャの一覧から使用できる。",""
-"★3確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。",""
-"SSR確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。",""
-"三女神様の祝福が得られるとされている。\nショップで様々なものと交換できる。",""
-"ウマ娘のやる気を絶好調にする。\nチーム競技場などで使用できる。",""
-"天候が「晴」、バ場が「良」になる。\nチーム競技場などで使用できる。",""
-"天候が「雨」、バ場が「不良」になる。\nチーム競技場などで使用できる。",""
-"枠番が内枠（1～3）になる。\nチーム競技場などで使用できる。",""
-"枠番が外枠（6～8）になる。\nチーム競技場などで使用できる。",""
-"リストの中から好きな\n★3育成ウマ娘と交換できる引換券。",""
-"リストの中から好きな\nSSRサポートカードと交換できる引換券。",""
-"持つべきものはフォロー数！\nフォロー枠を1つ増やすことができる。",""
-"サークルの団結の証。\nショップで様々なものと交換できる。",""
-"★2以上確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。",""
-"SR以上確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。",""
+"text", "translation</size>"
```

Comment:
you missed this one </size>

Reply:
Whoops. Me being lazy with mass replace.